rest-api.md

```diff
@@ -0,0 +1,767 @@
+## API
+
+### /keys
+
+**GET** `/v1/keys`
+
+Endpoint returns list of keys for all modules.
+
+Query:
+
+- `used` - filter for used/unused keys. Possible values: true/false;
+- `operatorIndex` - filter for keys of operator with index `operatorIndex`;
+
+```typescript
+interface ELBlockSnapshot {
+  blockNumber: number;
+  blockHash: string;
+  blockTimestamp: number;
+}
+
+interface Key {
+  index: number;
+  key: string;
+  depositSignature: string;
+  used: boolean;
+  operatorIndex: number;
+  moduleAddress: string;
+}
+
+class Response {
+  data: Key[];
+  meta: {
+    elBlockSnapshot: ELBlockSnapshot;
+  };
+}
+interface HttpException {
+  statusCode: number;
+  message: string;
+}
+
+class TooEarlyResponse implements HttpException {
+  statusCode: number = 425;
+  message: string = 'Too early response';
+}
+```
+
+Example:
+
+```
+curl -X 'GET' 'http://localhost:3000/v1/keys'
+```
+
+:::warning
+Response of this endpoint could be very large but we can’t have a pagination here since data could be updated in the process.
+:::
+
+:::warning
```

Comment:
Maybe write it in the beginning?
All requests return 425 until the service is completely initialized.